common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -176,6 +177,33 @@ public static BrokerResult brokerResultFromBundle(@NonNull final Bundle resultBu
                 );
     }
 
+    public static boolean getHelloResultFromBundle(final Bundle bundle) throws ClientException {
+        final String methodName = ":getHelloResultFromBundle";
+        if (bundle == null) {
+            Logger.warn(TAG + methodName, "The hello result bundle is null.");
+            return false;
+        } else if (!StringUtil.isEmpty(bundle.getString(AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY))) {
```

Comment:
No need of `else if` here as we are returning right away. Just using `if` will improve readability

Reply:
Personally, I always do early return wherever possible. (Also, if I can't do early return, it could be a sign that the function is doing 'too much').